apps/barista/src/components/icon-color-wheel/icon-color-wheel.ts

```diff
@@ -40,6 +41,16 @@ let uniqueColorwheelId = 0;
 export class BaIconColorWheel {
   @Input() iconname: string;
 
+  @Input()
+  get iconsvg(): string {
+    return this._iconsvg.toString();
+  }
+  set iconsvg(value: string) {
+    this._iconsvg = this._sanitizer.bypassSecurityTrustHtml(value);
+  }
+  /** @internal Icon SVG string as safe HTML */
+  _iconsvg: SafeHtml;
```

Comment:
Could this member be private and we use the getter in the template?
```suggestion
  private _iconsvg: SafeHtml;
```